Building in a docker container fails because of node-gyp --jobs max

I'm trying to install opencv4nodejs with opencv 4.3.0 in a docker container. I think it's failing because it's trying to run node-gyp using multiple cores which is not working inside the container for some reason. I fixed the issue by changing the `--jobs max` argument for node-gyp to `--jobs 1` on this line:

https://github.com/justadudewhohacks/opencv4nodejs/blob/master/install/install.js#L80

This is the Dockerfile for the container:

```
FROM node:14-bullseye

RUN apt-get update && \
    apt-get install -y build-essential && \
    apt-get install -y --no-install-recommends cmake && \
    rm -rf /var/lib/apt/lists/*

ENV OPENCV4NODEJS_AUTOBUILD_FLAGS="-DBUILD_LIST=core,imgproc,imgcodecs,videoio,highgui,video,calib3d,features2d,objdetect,dnn,ml,flann,photo,stitching,xfeatures2d"
ENV OPENCV4NODEJS_AUTOBUILD_OPENCV_VERSION="4.3.0"

# Enable verbose mode for node-gyp
# https://stackoverflow.com/a/19074181/545726
ENV V=1

WORKDIR /app
ADD package.json yarn.lock /app/

RUN yarn
```

# Error Message

There's no clear error message, it compiles a few files then after `Release/obj.target/opencv4nodejs/cc/core/HistAxes.o` it gets stuck then terminates:

```
g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
make: *** [opencv4nodejs.target.mk:242: Release/obj.target/opencv4nodejs/cc/opencv4nodejs.o] Error 1
make: *** Waiting for unfinished jobs....
make: Leaving directory '/app/node_modules/opencv4nodejs/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:400:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
gyp ERR! System Linux 5.10.25-linuxkit
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--jobs" "max"
gyp ERR! cwd /app/node_modules/opencv4nodejs
gyp ERR! node -v v14.17.6
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok
```

# Auto Build - Full Error Log

See attached full error log.
[20210923-opencv4nodejs-error-log.txt](https://github.com/justadudewhohacks/opencv4nodejs/files/7217526/20210923-opencv4nodejs-error-log.txt)

--- a/install/install.js
+++ b/install/install.js
@@ -77,8 +77,16 @@ process.env['OPENCV4NODEJS_DEFINES'] = defines.join('\n')
 process.env['OPENCV4NODEJS_INCLUDES'] = includes.join('\n')
 process.env['OPENCV4NODEJS_LIBRARIES'] = libs.join('\n')
 
-const flags = process.env.BINDINGS_DEBUG ? '--jobs max --debug' : '--jobs max'
-const nodegypCmd = 'node-gyp rebuild ' + flags
+const flags = [];
+if (process.env.BINDINGS_JOBS) {
+  flags.push('--jobs ' + process.env.BINDINGS_JOBS);
+} else {
+  flags.push('--jobs max');
+}
+if (process.env.BINDINGS_DEBUG) {
+  flags.push('--debug');
+}
+const nodegypCmd = 'node-gyp rebuild ' + flags.join(' ')
 log.info('install', `spawning node gyp process: ${nodegypCmd}`)
 const child = child_process.exec(nodegypCmd, { maxBuffer: Infinity }, function(err, stdout, stderr) {
   const _err = err || stderr

